.github/workflows/checks.yaml

```diff
@@ -56,3 +56,5 @@ jobs:
         run: |
           npm run seed:mock:all
           npm run build
+        env:
```

Comment:
doesn't this need to be set **before** `run`? 🤔

Reply:
Didn't think it mattered as long as it's in the same scope/indentation. But testing now. 👍🏻